example.env

```diff
@@ -0,0 +1,17 @@
+# Reference: https://github.com/frappe/frappe_docker/blob/main/docs/images-and-compose-files.md
```

Comment:
do any environment variables change?

they are used by containers here https://github.com/frappe/helm/tree/main/erpnext/templates in deployment-*.yaml

Reply:
They did. All of them are in [`example.env`](https://github.com/vrslev/frappe_docker/blob/beautify/example.env). Also, [docs](https://github.com/vrslev/frappe_docker/blob/beautify/docs/images-and-compose-files.md#configuration).

Reply:
is backward compatibility possible? I'll have to heavily change charts.

Reply:
It is already partially backwards compatible. Only variable name that changed is `DB_PASSWORD` (previously, it was `MYSQL_ROOT_PASSWORD` and `POSTGRES_PASSWORD`). It had to be done for Postgres override.

Other variables (`ADMIN_PASSWORD`, `MARIADB_HOST`, `ENTRYPOINT_LABEL` and similar) just not used anymore.